### ISSUE TITLE
Property:accessRights

### ISSUE THREAD

**Issue by mtrekels** [tdwg/cd#45]
|  |  |
| ---- | ---- |
| **Definition** | Information about who can access the resource or an indication of its security status. |
| **Dimension** |  |
| **Existing property** | dc:accessRights |
| **Existing class** |  |
| **Existing property identifier** | http://purl.org/dc/terms/accessRights |
| **Format** |  |
| **Required** |  |
| **Repeatable** |  |
| **Constraints** |  |
| **Examples** |  |
| **Notes** |  |

**Comment by mswoodburn** [comment]
Added some examples.

**Comment by magpiedin** [comment]
In the BBQ, we discussed what "resource" here (& in a few other fields, e.g. license) refers to.  "accessRights" here refers to access to the Collections Description record (which is consistent with the term's use in DwC)

Quentin recommends a separate physicalAccessRights in the Object Group Class, to cover access rights to the physical collection.

**Comment by wouteraddink** [comment]
I am not sure if this property is useful as part of a description? Having one for physical access rights makes sense but for this one it would mean that you send a full description record to an application to then discover that access is restricted, not very secure. Should that not be handled separate as part of operations you can perform on a record?

**Comment by qgroom** [comment]
> I am not sure if this property is useful as part of a description? Having one for physical access rights makes sense but for this one it would mean that you send a full description record to an application to then discover that access is restricted, not very secure. Should that not be handled separate as part of operations you can perform on a record?

Yes, I kind of feel the same way. I was only convinced by the analogy to Darwin Core that does the same thing. Though I think this doesn't make much sense in Darwin Core either.

**Comment by rondlg** [comment]
The accessRights term in dc seems to relate to the digital resource and in our case we want to allow the record creator to provide a link to information about access to the physical collection, suggest changing the property name to accessConditions.

**Comment by rondlg** [comment]
Move to ObjectGroup

**Comment by DavidFichtmueller** [comment]
Just to add a bit more details from todays discussion. This term is based on https://schema.org/conditionsOfAccess , with the difference that this is only meant for physical access, whereas the schema.org term is meant for both physical and digital access.
